### ISSUE TITLE
Setting "female" sometimes is different to male

### ISSUE THREAD

**Issue by Kodzhabashev** [humdek-unibe-ch/sh-selfhelp#159]
In GitLab by @olivierbur on Apr 5, 2019, 15:16

Sometimes, css-orders for the female setting is not carried out.

At the starting page of HERMES, there is a avatar displayed, daniel for males, maria for females https://selfhelp.psy.unibe.ch/hermes/  

The card where the images are in, is the same, thus both images should react in the same way. This is not the case. For males, daniel is displayed in the center ("text-center", CSS in the card), for females, it is displayed on the left side of the screen. Aren't the settings for females, male by default, if there is no specific "female" order?

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Apr 8, 2019, 11:40

mentioned in commit 2af6a1bd99bd51c893bdd9161330d7b34a1dff52

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Apr 8, 2019, 12:05

It looks like this was a major error on my part. The patch 2af6a1bd should fix this.

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Apr 9, 2019, 08:18

Version v1.0.4 is now deployed on HERMES.
The problem should be fixed now.
This was tested on a local copy.
Please verify this and close this issue if everything is ok.
Otherwise, don't hesitate to describe further problems here.

**Comment by Kodzhabashev** [comment]
In GitLab by @olivierbur on Apr 9, 2019, 09:45

It fixed one thing, that is the picture of "Maria" in the *Home* page ist now centered. 

However, several major errors still occur. For instance there is information displayed on this site https://selfhelp.psy.unibe.ch/hermes/PST that stems from the site that follows this one. So it is displayed twice. 

In other places, there is information missing. https://selfhelp.psy.unibe.ch/hermes/MeineEinstellungzuProblemen Here for instance, the conditional container does not have an input.

https://selfhelp.psy.unibe.ch/hermes/Problemdefinieren Or here, there is no title and some markdowns are missing.

**Comment by Kodzhabashev** [comment]
In GitLab by @olivierbur on Apr 9, 2019, 10:11

I am trying to figure out an underlying pattern, but I can't see any. It's hard to tell as some style are affected, whereas sometimes the same styles aren't

**Comment by Kodzhabashev** [comment]
In GitLab by @moiri on Apr 9, 2019, 10:31

I did find another bug and fixed it (deployed to the server).

The whole problem was at one point in the code so I have confidence that it does not impact other areas of the code. The problem was the following:

When fetching data from the database there is a check for the gender of the current user and the data according to the gender was fetched. The first problem was that if only some fields were defined for females, the system only fetched those fields and not the male fall-backs. The second problem was that the system did not check if a field was present in the DB but empty.

@olivierbur can you please check again and see whether things are ok?

**Comment by Kodzhabashev** [comment]
In GitLab by @olivierbur on Apr 9, 2019, 10:41

Great, thank you so much!

As far as I checked, things are ok now. There was just one more thing, a markdown text on an other page, that shouldn't be there, but I saw that the text was saved in the female markdown place, so I guess I must have add this wrongly, when I saw last friday that things were missing. 

I deleted this text and now it takes the male one, so this is fine!

I think that was it, thank you!
